pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -74,6 +74,7 @@ def __call__(self, value, system):
                 'imageFormat': 'image/png',
             }]
         }
+        basemap = extract_dict['baseLayers']['layers'][0]
```

Comment:
Shouldn't we create this variable directly, and use it in the `baseLayers` building...?

Reply:
Yeah, that is fine with me and probably the better solution! Do you adapt it or do you want me to do it?